src/components/Router.js

```diff
@@ -88,7 +88,7 @@ export default class Router extends Component {
         div(['Welcome to the new FireCloud interface, powered by Terra. All of your workspaces are available. ',
           link({
             ...Utils.newTabLinkProps,
-            href: 'https://broadinstitute.zendesk.com/hc/en-us/sections/360003528231-FireCloud-users-Find-out-what-s-new-in-Terra',
+            href: 'https://support.terra.bio/hc/en-us/sections/360003528231-FireCloud-users-Find-out-what-s-new-in-Terra',
```

Comment:
This is a broken link but the issue is the original link is broken, not the redirect. I'll find the correct new url and add a suggestion here, so please hold merging until we can fix this.